### PR TITLE
Implementation of District Colors, based on Province! #10

### PR DESCRIPTION
**Pull request by TheBinitGhimire**
Districts in the same province are now of same color.

### CODE REVIEW

File: components/District.tsx [hunk 1]
```diff
@@ -1,13 +1,18 @@
 type props = {
+  province: number;
   shape: string;
 };
 
-export default function District({ shape }: props) {
+const colors: string[] = Array(7)
+  .fill('#')
+  .map((c) => c + Math.random().toString(16).substring(2,8));
```
Comment (ashiishme) [inline components/District.tsx]: This could be utility function but I would rather go with RBG or HSL color generation so that we can adjust color.

**Review by ashiishme — CHANGES_REQUESTED** [review]

File: components/District.tsx (hunk 1 above)
Comment (TheBinitGhimire) [reply components/District.tsx]: How about now, generating HSL colors as per a1a3c81451d3ca596b0c7ede9d2607943239bb8f?

File: components/District.tsx [hunk 2]
```diff
@@ -1,13 +1,18 @@
 type props = {
+  province: number;
   shape: string;
 };
 
-export default function District({ shape }: props) {
+const colors: string[] = Array(7)
+  .fill('hsl(_)')
+  .map((c) => c.replace(/_/g, `${Math.floor(Math.random()*360)}, 100%, 50%`));
```
Comment (ashiishme) [inline components/District.tsx:8]: Can you extract this to a utility function file?

**Review by ashiishme — CHANGES_REQUESTED** [review]

File: components/District.tsx (hunk 2 above)
Comment (aabishkaryal) [reply components/District.tsx:8]: Maybe this is better?
```ts
const colors: string[] = Array(7)
  .map((_) => Math.floor(Math.random() * 360))
  .map((c) => `hsl(${c}, 100%, 50%)`);
```

Comment (aabishkaryal) [reply components/District.tsx:8]: > Maybe this is better?
> 
> ```ts
> const colors: string[] = Array(7)
>   .map((_) => Math.floor(Math.random() * 360))
>   .map((c) => `hsl(${c}, 100%, 50%)`);
> ```

This doesn't work apparently sorry

**Pull request merged**